src/atoms/gv-button.js

```diff
@@ -56,7 +56,7 @@ import { dispatchCustomEvent } from '../lib/events';
  * @cssprop {Color} [--gv-button-primary--c=var(--gv-theme-font-color-light, #ffffff)] - Primary color
  * @cssprop {Color} [--gv-button-primary--bgc=var(--gv-theme-color, #5a7684)] - Primary background color
  * @cssprop {Color} [--gv-button-danger--c=var(--gv-theme-font-color-light, #ffffff)] - Danger color
- * @cssprop {Color} [--gv-button-danger--bgc=var(--gv-theme-danger-color, #ff5722)] - Danger background color
+ * @cssprop {Color} [--gv-button-danger--bgc=var(--gv-theme-color-danger, #ff5722)] - Danger background color
```

Comment:
not sure but I think it can be a breaking change since variables can be overridden and saved to DB.  (see https://nightly.gravitee.io/api/management/organizations/DEFAULT/environments/DEFAULT/configuration/themes/default)

Reply:
The variable `--gv-theme-danger-color` couldn't have been overridden by clients as it was not part of the theme. In fact, before this fix, it was always falling back to the default value.